src/main/java/nus/climods/logic/LogicManager.java

```diff
@@ -36,13 +39,21 @@ public LogicManager(Model model, Storage storage) {
     }
 
     @Override
-    public CommandResult execute(String commandText) throws CommandException, ParseException {
+    public CommandResult execute(String commandText) throws CommandException, ParseException, IOException {
         logger.info("[User Command] " + commandText);
 
         CommandResult commandResult;
         Command command = CliModsParser.parseCommand(commandText);
         commandResult = command.execute(model);
 
+        switch (command.getCommandWord()) {
```

Comment:
I think it is a little awkward to have a `getCommandWord` for commands since it is already declared as a `public static`, putting all triggers (to save etc.) in `CommandResult` will solve this